app/subjects/mention-list.cjsx

```diff
@@ -35,13 +35,14 @@ module.exports = React.createClass
       <div>
         <h2>Mentions:</h2>
         <div>
-          {for mention, i in @state.mentions
+          {@state.mentions.map (mention) ->
             <Comment
-              key={"mention-#{mention.comment.id}-#{i}"}
+              key={"mention-#{mention.comment.id}"}
```

Comment:
Ah right. I've deleted that commit but I'm uneasy about the dependency on array order here. Using the array index as an identifier for each mention seems brittle to me. I guess it's okay since we only fetch the mentions array once from the API?